### DISCUSSION TITLE
how to change the character of this vertical line?

### DISCUSSION THREAD

**Discussion by hopezh** [lukas-reineke/indent-blankline.nvim#859] [Q&A] [answered]
As shown below, which option should I use to change the character for this vertical line highlighting the current context?

![2024-04-17 23 10 45](https://github.com/lukas-reineke/indent-blankline.nvim/assets/23435810/08add998-32bc-4b5e-b036-69fd1c408ddf)

**Comment by lukas-reineke** [comment, marked as answer]
It's `:help ibl.config.scope.char`

**Reply by hopezh** [reply]
I used the following config of `char=":"`, but the character representing the vertical line doesn't seem to change. Have I missed anything important here? 

![2024-04-18 10 12 00](https://github.com/lukas-reineke/indent-blankline.nvim/assets/23435810/ae3e04c8-205a-4811-821f-6e33279fc6ae)

**Reply by lukas-reineke** [reply]
scope char without indent char is currently not supported.
